.azuredevops/containerApp.yaml

```diff
@@ -8,7 +8,7 @@ properties:
   managedEnvironmentId: {{managedEnvironmentId}}
   configuration:
     ingress:
-      external: false
+      external: true
```

Comment:
just to confirm this is 'VNET Only' and not 'fully external'. I know it had to change from only being 'Container Apps Only' but should be only accessible within the Network.